src/main/java/microservices/userServiceBasic/repository/model/UserDto.java

```diff
@@ -0,0 +1,102 @@
+package microservices.userServiceBasic.repository.model;
+
+import microservices.utils.emailServices;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.Id;
+
+@Entity
+public class UserDto {
+
+	private @Id
+	@GeneratedValue
+	long id;
```

Comment:
id should be renamed to uuid and to be UUID type. Remove all the JPA annotations for now.